vmdb/app/models/ems_openstack_infra.rb

```diff
@@ -1,6 +1,8 @@
 class EmsOpenstackInfra < EmsInfra
   include EmsOpenstackMixin
 
+  has_many :orchestration_stacks, :foreign_key => :ems_id, :dependent => :destroy
```

Comment:
I see the potential for moving this up to EmsOpenstackMixin. I guess that's what @bzwei is talking about in the comments in the main conversation.

I'm fine for it being in Infra for now, then later finding its way to the Mixin when the cloudy orchestration bits are added.